src/__tests__/utils/isObjectLike.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Проверяет, является ли `value` похожим на объект. Значение считается похожим на объект, если оно не `null` и имеет результат `typeof` равный "object".
+ *
+ * @param {any} value - Значение для проверки.
+ * @returns {boolean} Возвращает `true`, если `value` похоже на объект, иначе `false`.
+ */
+export function isObjectLike(value: any) {
```

Comment:
тут не уверен, но можно попробовать тоже добавить:

```suggestion
export function isObjectLike(value: any): value is object {
```